pom.xml

```diff
@@ -223,6 +223,14 @@
                 <scope>import</scope>
             </dependency>
 
+            <dependency>
+                <groupId>com.oracle.database.jdbc</groupId>
+                <artifactId>ojdbc10-production</artifactId>
+                <version>19.21.0.0</version>
```

Comment:
Why not use `${dep.oracle.version}`?

Reply:
I'll remove the property as there is a single usage